runtime/http_server_apis.md

```diff
@@ -256,3 +256,48 @@ Currently Deno does not support upgrading a plain-text HTTP/1.1 connection to an
 HTTP/2 cleartext connection via the `Upgrade` header (see:
 [#10275](https://github.com/denoland/deno/issues/10275)), so therefore HTTP/2
 support is only available via a TLS/HTTPS connection.
+
+### Serving WebSockets
+
+Deno can upgrade incoming HTTP requests to a WebSocket. This allows you to
+handle WebSocket endpoints on your HTTP servers.
+
+To upgrade an incoming `Request` to a WebSocket you use the
+`Deno.upgradeWebSocket` function. This returns an object consisting of a
+`Response` and a web standard `WebSocket` object. The returned response should
+be used to respond to the incoming request using the `respondWith` method. Only
```

Comment:
I'm a little bit confused by this sentence, I don't see any calls to the `respondWith` method in the example code below. It may be helpful to add an example of using the `respondWith` method.